src/app/components/Icons/FavoritesIcon.tsx

```diff
@@ -0,0 +1,13 @@
+import SVG, { SVGProps } from '../IconSvg/IconSvg';
+
+function FavoritesIcon(props: Omit<SVGProps, 'children'>): JSX.Element {
+  return (
+    <SVG {...props}>
+      <path d="M0,0h24v24H0V0z" fill="none" />
+      <path d="M0,0h24v24H0V0z" fill="none" />
```

Comment:
Are these required because they might be invisible?

Reply:
yes, because of the hole in the icon itself, like a cutout